src/data/index.js

```diff
@@ -1820,4 +1820,52 @@ export default [
       'Bloomberg',
     ],
   },
+  {
+    id: 164,
+    name: 'Longest Word in Dictionary',
+    url: 'https://leetcode.com/problems/longest-word-in-dictionary/',
+    pattern: 'Trie',
+    difficulty: 'Easy',
+    companies: [],
+  },
+  {
+    id: 165,
+    name: 'Maximum XOR of Two Numbers in an Array',
+    url: 'https://leetcode.com/problems/maximum-xor-of-two-numbers-in-an-array',
+    pattern: 'Trie',
+    difficulty: 'Medium',
+    companies: [],
+  },
+  {
+    id: 166,
+    name: 'Concatenated Words',
+    url: 'https://leetcode.com/problems/concatenated-words/',
+    pattern: 'Trie',
+    difficulty: 'Hard',
+    companies: [],
```

Comment:
The companies for `'Concatenated Words'` are: `'Amazon', 'Apple', 'Facebook'` (_I don't include lesser known companies like Audible_)

![image](https://user-images.githubusercontent.com/13009507/82843221-7f901780-9eaa-11ea-958d-6f34ac9d4a08.png)